File: corpus/py/F;0543.py
Scanner entrada = new Scanner (System.in)
int npos, n
print("Digite um numero n positivo entre 2 e 18: ")
npos = int(input())
print("Digite o tamanho desejado da matriz n x n: ")
n = int(input())
for i=0 in range(0, n):
    for j=0 in range(0, n):
        print("Digite o número de milhos na posição i" + (i +1)+ " e posição j"+ (j +1))
for i=0 in range(0, n):
    print(" ")
    for j=0 in range(0, n):
        print(A[i][j] + " ")

File: corpus/2018q3/_p2q2_ok/_check/F;0543.java

import java.util.Scanner;

public class Tres {

    public static void main(String[] args) {

        Scanner entrada = new Scanner (System.in);

        int npos, n;

        System.out.print("Digite um numero n positivo entre 2 e 18: ");
        npos = entrada.nextInt();
        System.out.print("Digite o tamanho desejado da matriz n x n: ");
        n = entrada.nextInt();
        int A[][] = new int [n][n];

        for (int i=0; i<n; i++){
            for (int j=0; j<n; j++){
                System.out.println("Digite o número de milhos na posição i" + (i+1)+ " e posição j"+ (j+1));
                A[i][j] = entrada.nextInt();
            }
        }
        for (int i=0; i<n; i++){
            System.out.println(" ");
            for (int j=0; j<n; j++){
                System.out.print(A[i][j] + " ");
            }
        }

        }

    }
